### PR TITLE
Change site name

### LINKED ISSUE #225: Change site name
## Description
The new name of the Nextdoor Nature site is:

Nextdoor Nature Hub / Hwb Natur Drws-nesaf

## Acceptance Criteria

- [ ] As per plan

## Plan

- [ ] Replace "The Nextdoor Nature Library" with "Nextdoor Nature Hub" and the equivalent Welsh
- [ ] Replace "The Nextdoor Nature online library is a project..." with "Nextdoor Nature Hub is a project"
- [ ] Check (with Emma?) if it seems appropriate to sway library for hub in other places of our temp copy.

Other occurrences of "library"
- [ ] Explore the library --> Explore the hub
- [ ] This online library aims to support --> This online hub aims to support
- [ ] Search the library --> Search the hub

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #225 

## Description

- Nextdoor Nature Library -> Nextdoor Nature Hub
- (online) Library -> Hub
- Welsh language name

@geeksforsocialchange/developers

### CODE REVIEW

**Comment by katjam** [comment]
Can we checkout the content files so that this PR is only the intentional changes?

We'll do https://github.com/geeksforsocialchange/teamwilder/issues/280 to make sure it doesn't happen again in future.

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>[cCc] Nextdoor Nature Guides</title>
+    <title>[cCc] Nextdoor Nature Hub</title>
```
Comment (r-ferrier) [inline index.html:7]: Should this have been changed? It does seem inconsistent with everything else but is there a reason for that?

**Review by r-ferrier — COMMENTED** [review]
One q about the change to 'nextdoor nature guides' in the header - couldn't see this mentioned in the issue but as it is [cCc] copy anyway it's probably just been missed and is fine to also change?

**Review by r-ferrier — APPROVED** [review]
I'm approving because it's the page title so it seems very likely to me this was just missed off the issue in error!

**Pull request merged**